cfme/infrastructure/host.py

```diff
@@ -607,29 +608,25 @@ def find_quadicon(host, do_not_navigate=False):
     """Find and return a quadicon belonging to a specific host
 
     Args:
-        host: Host name as displayed at the quadicon
+        host: Host object
```

Comment:
Have you audited the codebase for all uses of this find_quadicon, dynamic and static, since this change will break existing calls?